### ISSUE TITLE
How to understand that new chunk is loaded?

### ISSUE THREAD

**Issue by Favkis** [Zylann/godot_voxel#179]
I am using binaries from march.

So module doesn't remember modifications by itself and they have to be written. I did that, I save all modifications, but I cannot get how to understand that new chunk has been loaded and I need to apply modifications made in this chunk.

**Comment by Zylann** [comment]
As you said in #178 the binaries from march don't have the signal telling you when a block is loaded/unloaded. This is a recent feature.

**Comment by Favkis** [comment]
Yes, just wanted to know if there's a way. I'll try to compile for Win10 and if it works - I'll use newer version, if not, I already came up with idea. I manage blocks from GDscript and by knowing render distance and my position I know which blocks needs to be laoded so I just force load one voxel, checking if block applied, if yes - then block is loaded and I proceel loading all remaining voxels.
